code/style.css

```diff
@@ -71,6 +80,8 @@ select {
   flex-direction: column;
   align-items: center;
   justify-content: space-between;
+  box-shadow: 0 4px 8px 0 rgb(0 0 0 / 20%), 
+  0 6px 20px 0 rgb(0 0 0 / 19%);
```

Comment:
This is such a cool styling effect to make the cards stick out and kind of float on the board.